### ISSUE TITLE
question about multi scale training

### ISSUE THREAD

**Issue by freaad** [clovaai/c3_sinet#1]
Thank you for sharing your code.

I have read about your multi-scale training code.
I am wondering it gives better results?
Where do you get the idea about this code?
How the loss acts when you do multi-scale training?
Thank you.
